File: 최수환-suwhan2/0501/callBy.py
'''
1. 정수 연산하기
    1. 인자 두 정수를 비교해 큰값에 *2
    2. 작은값에 +10
    3. 계산결과 반환
2. 정수 두개 입력받기
3. 받은 정수 연산하기
4. 출력
'''

def calculate(a,b) :
    if a>b :
        a*=2
        b+=10
    else:
        a+=10
        b*=2
    return a,b
a ,b= input().split(' ')
a,b = calculate(int(a),int(b))

print(a,b)

File: 임지훈-vanillacake369/Lv2/CallBy.java
import java.util.Scanner;

public class CallBy {
    public static void main(String[] args) {
        /* input */
        Scanner scanner = new Scanner(System.in);
        String[] inputStr = scanner.nextLine().split(" ");
        int a = Integer.parseInt(inputStr[0]);
        int b = Integer.parseInt(inputStr[1]);

        /* calc */
        int[] result = computeTwoInt(a, b);

        /* result */
        System.out.println((result[0]) + " " + (result[1]));
    }

    /**
     * 의사코드
     * 1. 두 개짜리의 배열 생성
     * 2. 입력 a,b 비교
     * 2-1. a가 크다면 :: 배열의 0 번째에 a*2, 1번째에 b+10 저장
     * 2-2. b가 크다면 :: 배열의 0 번째에 a+10, 1번째에 b*2 저장
     * 3. 배열 반환
     */
    private static int[] computeTwoInt(int a, int b) {
        int[] temp = new int[2];

        if (a > b) {
            temp[0] = a * 2;
            temp[1] = b + 10;
        } else {
            temp[0] = a + 10;
            temp[1] = b * 2;
        }

        return temp;
    }
}
